groups/sig-cluster-lifecycle/groups.yaml

```diff
@@ -1,5 +1,4 @@
 groups:
-
```

Comment:
I know this should not make a difference, but from a quick look it seems that all groups files have those empty lines, can we leave them?

Reply:
Ahhhh, format on save 😢  Yes no problem, sorry about that.

Reply:
Fixed the erroneous formatting.